assets/js/admin/marketplace-suggestions.js

```diff
@@ -130,6 +130,19 @@
 			return image;
 		}
 
+		// Render DOM element for suggestion icon image.
+		function renderSuggestionSvgIcon( slug, svgContent ) {
```

Comment:
slug is never used inside this function, any specific reason for including it?

Reply:
Good spotting – I can fix that on the main PR.